### DISCUSSION TITLE
overlaying actual observed drifter tracks

### DISCUSSION THREAD

**Discussion by jamespatrickmanning** [OpenDrift/opendrift#1147] [Q&A]
I like the OpenDrift plots of model trajectories but I would like to be able to overlay real drifter tracks.

I tried following simple sequence:

fig = plt.figure()
ax = fig.add_subplot(1, 1, 1)
OD.plot()
ax.plot(drifterlons,drifterlats)

but that puts the modelled and observed tracks in separate figures.

**Comment by knutfrode** [comment]
One option is to add a dictionary `drifter` like in the last codeline of this example:
https://opendrift.github.io/gallery/example_current_from_drifter.html

More flexibility is available if using the separate TrajAn package (under development):
https://opendrift.github.io/trajan/gallery/

But, I believe that your approach should work if you do it like this:
```
ax, fig = o.plot()
import cartopy.crs as ccrs
ax.plot(drifterlons, drifterlats, transform=ccrs.PlateCarree(globe=crs.globe))
```
